vendor-extra/LiberoPageBundle/src/Controller/PageController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPageBundle\Controller;
+
+use Libero\LiberoPageBundle\Event\CreatePageEvent;
+use Libero\LiberoPageBundle\Event\LoadPageEvent;
+use Libero\LiberoPageBundle\Exception\NoContentSet;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Twig\Environment;
+use function count;
+use function Libero\LiberoPageBundle\text_direction;
+
+final class PageController
+{
+    private $dispatcher;
+    private $template;
+    private $twig;
+
+    public function __construct(Environment $twig, string $template, EventDispatcherInterface $dispatcher)
+    {
+        $this->twig = $twig;
+        $this->template = $template;
+        $this->dispatcher = $dispatcher;
+    }
+
+    public function __invoke(Request $request) : Response
+    {
+        $context = [
+            'lang' => $request->getLocale(),
+            'dir' => text_direction($request->getLocale()),
+        ];
+
+        $loadEvent = new LoadPageEvent($request, $context);
```

Comment:
from the meeting: could be `LoadDataPageEvent` to avoid the lack of an object in the phrase; `Load...PageEvent` in general